### PR TITLE
switch base image from alpine to ubuntu

### PR DESCRIPTION
**Pull request by curtisbeck**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-GitPython==3.1.8
-pytest==6.0.2
-tox==3.20.0
+GitPython==3.1.11
+psycopg2-binary
```
Comment (system1-dersek) [inline requirements.txt]: Would it be better to leave this requirement to the tox environment?

Also with the binary package, it sounds like we shouldn't need pg_config - should we switch this to the normal psycopg2 if we're including libpq-dev, or have we tried removing libpq-dev with the binary package?

Comment (curtisbeck) [reply requirements.txt]: I thought I had removed this. I'll rebuild locally and test again.

Comment (curtisbeck) [reply requirements.txt]: removed it

**Comment by bra-fsn** [comment]
Thanks!

**Pull request merged**